jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -730,215 +730,6 @@ properties:
             type: number
             fits_keyword: PA_APER
             fits_hdu: SCI
```

Comment:
What do you want to do in the case where one keyword in a group is written to the SCI extension, but another is not? For example, in velocity_aberration va_scale is written to the SCI extension, but ra_offset and dec_offset are not. Should all the properties under velocity_aberration be copied into wcsinfo.schema.yaml or only the one keyword, va_scale?